### DISCUSSION TITLE
Command line shows when ale_hover_cursor message is called

### DISCUSSION THREAD

**Discussion by babakou9** [w0rp/ale#4660] [Q&A - Ask for help with problems]
Hey there,

I am using Ale on neovim with Folke/noice plugin to manage messages and command line.
when I hover on a line where Ale shows a lint error the message is showed in a popup as expected, but it shows the command during a fraction of a second and it is a bit disturbing.

![pb_ale](https://github.com/dense-analysis/ale/assets/66430862/2fcdba17-b991-477c-8648-4a210664a4dc)

I think I found the origin of the command, in ale/cursor.vim, but I have no idea how to change it in order to call the message without showing the prompt...

Anyone has an idea ?

**Comment by k4yt3x** [comment]
I just ran into the same problem. Opened an issue here: https://github.com/dense-analysis/ale/issues/4832
